File: userinput/userinput.java
import java.util.Scanner;
public class userinput {
	public static void main(String args[]) {
		System.out.print("Input: ");
		Scanner scan = new Scanner(System.in);
		String input = scan.nextLine();
		scan.close();
		System.out.println("You entered, \"" + input + "\"");
	}
}

File: UserInput/UserInput.c
#include <stdio.h>
#include <string.h>

#define MAX 100

int main(int argc, char *argv[]) {
	char input[MAX] = "String";
	printf("Input: ");
	fgets(input, sizeof input, stdin);
	if ((strlen(input) > 0) && (input[strlen(input) -1] == '\n')) {
		input[strlen(input) -1] = 0;
    }
	printf("You entered, \"%s\"\n", input);
	return 0;
}
